torch/distributed/c10d_error_logger.py

```diff
@@ -11,23 +11,20 @@
 
 from torch.distributed.logging_handlers import _log_handlers
 
-_c10d_error_logger = None
+__all__ = ["_get_or_create_logger", "_get_logging_handler"]
```

Comment:
I don't think we want to make these functions public, if you set __all__ to empty array then I think the test would still pass.

```suggestion
__all__ = []
```

Reply:
Ye. It's in V14, forgot to amend and push earlier.